python/ray/data/tests/test_parquet.py

```diff
@@ -393,10 +393,10 @@ def test_parquet_read_bulk(ray_start_regular_shared, fs, data_path):
     assert "test1.parquet" in str(input_files)
     assert "test2.parquet" in str(input_files)
     assert not ds._plan.has_started_execution
+    assert ds.schema() == Schema(pa.schema({"one": pa.int64(), "two": pa.string()}))
+    assert not ds._plan.has_started_execution
```

Comment:
I think we should just remove this assertion. This method call does trigger execution, but it's not captured by `ds._plan.has_started_execution`.

In this test (and `test_parquet_read_bulk_meta_provider`), we go down this code path: https://github.com/ray-project/ray/blob/0dfd933a58b2eed4475054aef5c403619711017e/python/ray/data/dataset.py#L3064-L3065. Since we create a new dataset (with `limit(1)`), that dataset starts execution, but not the original.
 
```suggestion
```